libs/extensions/src/constants/index.ts

```diff
@@ -6,3 +6,33 @@ export const ConstantControllers: { [key: string]: ControllerFunction } = {
 };
 
 export const PROTECTED_SETTINGS = 'PROTECTED';
+
+export { RSE, RSERRORS } from './errors';
+export * from './events';
+
+//For Ability Guard
+export const ACTIONS = {
+  MANAGE: 'manage',
+  CREATE: 'create',
+  UPDATE: 'update',
+  DELETE: 'delete',
+  READ: 'read',
+};
+
+// For Ability Guard
+export const SUBJECTS = {
```

Comment:
Each library should publish their own Subjects.

Reply:
This was done to integrate the JWT Guard Service to be used for any module and remove the circular dependency between @rumsan/extensions and @rumsan/user libraries.